src/darker/tests/test_main.py

```diff
@@ -929,3 +937,22 @@ def test_stdout_path_resolution(git_repo, capsys):
 
     assert result == 0
     assert capsys.readouterr().out == 'print("foo")\n'
+
+
+@pytest.mark.skipif(not sys.platform.startswith("win"), reason="Argument length limit on Windows")
```

Comment:
This line is too long, Black wants to reformat it as:
```suggestion
@pytest.mark.skipif(
    not sys.platform.startswith("win"), reason="Argument length limit on Windows"
)
```